app/styles/app.css

```diff
@@ -24,6 +37,17 @@ body {
   padding: 0;
 }
 
+html, body {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto', sans-serif, serif;
+  font-size: 15px;
+  height: 100%;
+  background-color: var(--colorBackground);
+  color: var(--colorText);
+  min-height: 100vh;
+}
```

Comment:
Let's please keep html, body styles on the top, since they're at the top of the DOM tree hierarchy